pkg/capabilities/consensus/datafeeds/feeds_aggregator.go

```diff
@@ -0,0 +1,137 @@
+package datafeeds
+
+import (
+	"math"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/shopspring/decimal"
+	ocrcommon "github.com/smartcontractkit/libocr/commontypes"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/types"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/mercury"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+type AggregatorConfig struct {
+	Tickers map[string]TickerConfig
```

Comment:
Could we use feeds here, too, to match the mercury report output? Namely, we should consistently refer to Feed IDs, which will be uint64 (https://github.com/smartcontractkit/chainlink/pull/11901)